cypress/e2e/tests/migration/applicationinventory/applications/manage_credentials_many_apps.test.ts

```diff
@@ -0,0 +1,138 @@
+/*
+Copyright © 2021 the Konveyor Contributors (https://konveyor.io/)
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+/// <reference types="cypress" />
+
+import {
+    login,
+    getRandomApplicationData,
+    getRandomAnalysisData,
+    deleteByList,
+    manageCredentialsForMultipleApplications,
+    writeMavenSettingsFile,
+} from "../../../../../utils/utils";
+import { Analysis } from "../../../../models/migration/applicationinventory/analysis";
+import { CredentialType, SEC, UserCredentials } from "../../../../types/constants";
+import * as data from "../../../../../utils/data_utils";
+import { CredentialsSourceControlUsername } from "../../../../models/administration/credentials/credentialsSourceControlUsername";
+import { CredentialsMaven } from "../../../../models/administration/credentials/credentialsMaven";
+import { Application } from "../../../../models/migration/applicationinventory/application";
+let source_credential: CredentialsSourceControlUsername;
+let maven_credential: CredentialsMaven;
+let sourceApplicationsList: Array<Analysis> = [];
+let mavenApplicationsList: Array<Analysis> = [];
```

Comment:
**nitpick:** These two arrays can be declared as `const`

Reply:
Fixed